.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @storageos/controlsurface
```

Comment:
Not sure if the error will clear up after merging but I have double checked it's correct and the team exists thus I am expecting that to be the case here